tekton/release-cheat-sheet.md

```diff
@@ -144,7 +144,9 @@ the pipelines repo, a terminal window and a text editor.
    (This can be done on the Github UI.)
    Make sure to fetch the commit specified in `TEKTON_RELEASE_GIT_SHA` to create the released branch.
 
-1. Edit `README.md` on `main` branch, add entry to docs table with latest release links.
+1. Edit `README.md` on `main` branch, add entry to docs table with latest release links. 
+
+1. Edit `releases.md` on the `main` branch, add an entry for the release.
```

Comment:
After adding an entry to `releases.md`, do release manager need to take any further action on the older entries?

Reply:
Yes, most likely. 
I am the release manager for v0.41.0 which will be the first LTS release and the first release for which the new rule will apply, so as part of v0.41.0 I will update this doc and update the cheat-sheet as required if that's ok?

Reply:
yup that will work 👍